workflows/files/ZWECONF.xml

```diff
@@ -480,19 +480,52 @@ How we want to verify SSL certificates of services. Valid values are:
             <default>false</default>
         </boolean>
     </variable>
```

Comment:
also there on line 472, forgotten to remove this obsolete variable definition from the workflow, components_gateway_debug, when changed to components_zaas_debug